### ISSUE TITLE
Schedule Notification 

### ISSUE THREAD

**Issue by Somtobro** [MaikuB/flutter_local_notifications#1813]
Hello guys...the schedule notification for me only gets called the first day and after that it doesn't get called
I'm using version 12.0.0 of the plug in and flutter 3.3

Also I think I configured the local time and set it properly 10:03 daily.....
It works the first day and doesn't work anymore

The error might be from me ...anyways if you have any idea on what might be wrong let me know 👍🏾

**Comment by MaikuB** [comment]
Which device(s) did you test this on? If it's an Android device then the issue you've come across would relate to what's mentioned at https://github.com/MaikuB/flutter_local_notifications/tree/master/flutter_local_notifications#scheduled-android-notifications. I retested this on my Pixel 6 as this would be like a stock/pure Android OS implementation and it works as expected. If this is to do with the Android device then there's nothing further the plugin can do

**Comment by Somtobro** [comment]
Hey
In the example app, I can't seem to understand how the scheduleDailyTenAm()
function works.
From what I can see it is called only when the button is pressed.
But how does it set for the next day if it is called only when the button
is pressed?
Or is it called again?
I would appreciate your help please

**Comment by Somtobro** [comment]
This is a continuation
It is also worthy of note to know that
Whenever I add just 30 seconds from the current time to the schedule
notification. It works fine.
But I when I set 10 am it doesn't work.
I even copied and pasted the nextInstanceTenAm() function to set the next
date.
It's a weird issue and it's frustrating because I have to wait a whole day
to test it and I've been on it for weeks now
👍🏾Please I need your opinion desperately
And other notifications work well

**Comment by novas1r1** [comment]
@Somtobro just as a hint: you dont have to wait the whole day. Just change the system date in your emulator. That works fine for testing for me.

**Comment by Somtobro** [comment]
@novas1r1 Thank you very much.
I will try that
Even though I use my real device to test but I don't mind honestly🤡

**Comment by MaikuB** [comment]
> From what I can see it is called only when the button is pressed.
But how does it set for the next day if it is called only when the button
is pressed?
Or is it called again?
I would appreciate your help please

Reminder you still haven't answered my previous question on which devices your are using. Putting that aside, to answer this, this is already explained in the API docs and readme. Please read https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/FlutterLocalNotificationsPlugin/zonedSchedule.html. I believe if you read this properly then you will understand how it works properly that it also explains why you may think it doesn't work

**Comment by Somtobro** [comment]
Oh
I tested on Vivo and Infinix devices

**Comment by Somtobro** [comment]
Thanks for checking in and also the doc
I just realized that you don't need to set the daily notification everyday,
the matchdate... Parameter does that for you. Cool
But then, the content of my notification is different everyday
How do you suggest I go about it? in order to not set the notification
everyday cos personally I don't even like doing that I think the approach
you guys used it's very creative.

**Comment by MaikuB** [comment]
Thanks for the compliments but it's really just using what the platform's native notification APIs provide as that is the purpose of a plugin. There's no such thing to do what you want via these APIs. One potential solution that comes to mind is run a recurring background job that does display a different notification each day. This is something you'll need to do more research on to solve. Closing this as I've explained the docs and what you're trying to do isn't available via notification APIs and is requires more research on your part

**Comment by Somtobro** [comment]
Fair enough

**Comment by Somtobro** [comment]
Hello, I figured out the root of all the issues I was having (why the
notifications were not showing up).it's wasn't even about dynamic data or
anything trying to do some background tasks before setting the notification.

      The issue was caused by the big picture...I stored some images in
getApplicationDocumentsDirectory()/image.png from the shared preference
plugin
When I set the image in the 1st parameter of the BigPictureStyleInformation
class. It doesn't work sometimes, sometimes it works too
I don't know why though

Also note that the image.png I set is dynamic it changed whenever the app
opens.it may be why it happens don't know

It's actually an interesting issue just thought to let you know.
Also is there a way to render latex instead of just Strings in the
notification title or body?
